content/en/docs/pipelines/sdk/static-type-checking.md

```diff
@@ -141,12 +159,16 @@ when the upstream component generates an output with type "*Float*" and the down
 "*Float*" or "*Integer*", it might fail if you define the type as "*Float_or_Integer*". 
 Disabling the type checking per-argument is also supported as shown below.
 ```python
-@dsl.pipeline(name='type_check_a',
-    description='')
+@dsl.pipeline(name='type_check_a', description='')
 def pipeline():
-    a = task_factory_a(field_l=12)
-    # For each of the arguments, you can also ignore the types by calling ignore_type function.
-    b = task_factory_b(field_x=a.outputs['field_n'], field_y=a.outputs['field_o'], field_z=a.outputs['field_m'].ignore_type())
+  a = task_factory_a(field_l=12)
+  # For each of the arguments, you can also ignore the types by calling
+  # ignore_type function.
+  b = task_factory_b(
+      field_x=a.outputs['field_n'],
+      field_y=a.outputs['field_o'],
+      field_z=a.outputs['field_m'].ignore_type())
+
 compiler.Compiler().compile(pipeline, 'pipeline.tar.gz', type_check=True)
 ```
 
```

Comment:
Suggestion:

- Line 183: To provide more information, making it explicit that it's a Jupyter-based demo, and to improve accessibility, perhaps we can slightly rephrase this to:

```diff
- See [type checking sample]
+ Learn how to define a KubeFlow pipeline with Python DSL and compile the
+ pipeline with type checking: a [Jupyter notebook 
+ demo](https://github.com/kubeflow/pipelines/blob/master/samples/core/dsl_static_type_checking/dsl_static_type_checking.ipynb)
```

Or something along these lines.

WDYT?

Reply:
SG. Done